config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  # Defines the root path route ("/")  
+  #root "users#index"
+  
+  get "/users/:id/posts", to: "posts#index"
+  get "/users/:id/posts/:id", to: "posts#show"
```

Comment:
- [ ] The route to the posts#show will cause an error in the long run because you have two params called `:id`. The first one should be the `user_id` while the second should be `:id`. Kindly change it. 👍 😎 

#### Hint: `get "/users/:user_id/posts/:id", to: "posts#show"`